app/components/multi-select-dropdown.component.scss

```diff
@@ -0,0 +1,57 @@
+@import 'vars';
+
+.dropdown-header {
+    color: $conversationDarkGray;
+    font-weight: 700;
+}
```

Comment:
we should avoid any non-specific classes, we don't want to accidentally override styles on a site importing these classes. Also it doesn't look like this is being used.